Bitkit/Components/NumberPadTextField.swift

```diff
@@ -74,24 +74,33 @@ struct NumberPadTextField: View {
 
     @ViewBuilder
     private var primaryDisplayView: some View {
+        let font = Font.custom(Fonts.black, size: 44)
+        let placeholder = viewModel.getPlaceholder(currency: currency)
+
+        let valueText: Text = {
+            if !viewModel.displayText.isEmpty {
+                let entered = Text(viewModel.displayText)
+                    .font(font)
+                    .foregroundColor(.textPrimary)
+                let remainder = Text(placeholder)
+                    .font(font)
+                    .foregroundColor(isFocused ? .textSecondary : .textPrimary)
+                return entered + remainder
```

Comment:
Does the `+` result in a single view in the UI tree?

Reply:
Yes, indeed.`+` operator on `Text` values returns another `Text`.